wof/examples/flask/odm2/timeseries/odm2_timeseries_dao.py

```diff
@@ -1,57 +1,52 @@
 from __future__ import (absolute_import, division, print_function)
 
-from datetime import datetime
+from dateutil.parser import parse
+
 from sqlalchemy import create_engine
 from sqlalchemy.orm import scoped_session, sessionmaker
-from sqlalchemy.sql import and_
-
-from dateutil.parser import parse
 
 from wof.dao import BaseDao
 import wof.examples.flask.odm2.timeseries.sqlalch_odm2_models as model
-from odm2api.ODM2 import models as odm2_models
-from sqlalchemy.orm import aliased
-from sqlalchemy import or_
-from sqlalchemy import and_
-from sqlalchemy.sql import func
 
-from sqlalchemy.orm import with_polymorphic
+from odm2api.ODM2 import models as odm2_models
```

Comment:
Are these two statements actually referring to different things?
```python
# line 9
import wof.examples.flask.odm2.timeseries.sqlalch_odm2_models as model
# vs
from odm2api.ODM2 import models as odm2_models
```

If they're the same, how about "merging" them?